### ISSUE TITLE
Cannot transfer funds between spot and main accounts on Bibox.

### ISSUE THREAD

**Issue by karlandoh** [ccxt/ccxt#5931]
- OS: Linux
- Programming Language version: Python3
- CCXT version: Latest
- Exchange: Bibox
- Method: ```privatePostTransfer```
```
bibox.privatePostTransfer({'currency': 'BTC', 'amount': 3.685e-05, 'type': 1, 'version': 'v2'})

Request: POST https://api.bibox.com/v1/transfer {'Content-Type': 'application/json', 'User-Agent': 'python-requests/2.19.1', 'Accept-Encoding': 'gzip, deflate'} {"cmds":"[{\"currency\":\"BTC\",\"amount\":3.685e-05,\"type\":1,\"version\":\"v2\"}]","apikey":"PERSONALLY OMITTED","sign":"b14bce1fd343e23c99ef5c012909a969"}

Response: POST https://api.bibox.com/v1/transfer 200 {'Server': 'Tengine', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '67', 'Access-Control-Allow-Origin': 'undefined', 'Access-Control-Allow-Methods': 'POST, GET, OPTIONS', 'Access-Control-Allow-Headers': 'Content-Type, Accept, x-access-token, X-Requested-With', 'Access-Control-Max-Age': '172800', 'X-Ca-Request-Id': '530F41F2-424A-499C-B46B-0FFBD741DA92', 'X-Powered-By': 'Express', 'Access-Control-Allow-Credentials': 'true', 'ETag': 'W/"43-ZqeOEf4ZLo+6gjGn1D3me1vXNco"', 'Date': 'Sun, 06 Oct 2019 21:48:31 GMT', 'Connection': 'keep-alive'} {"error":{"code":"3011","msg":"接口不支持apikey请求方式"}}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 436, in inner
    return entry(_self, **inner_kwargs)
  File "/usr/local/lib/python3.7/site-packages/ccxt/bibox.py", line 861, in request
    response = self.fetch2(path, api, method, params, headers, body)
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 455, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 563, in fetch
    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)
  File "/usr/local/lib/python3.7/site-packages/ccxt/bibox.py", line 855, in handle_errors
    raise ExchangeError(feedback)
ccxt.base.errors.ExchangeError: bibox {"error":{"code":"3011","msg":"接口不支持apikey请求方式"}}
```

Strangely, this issue arises. However, I am able to fetch my balances just fine! Meaning that my apiKey and secret parameters are in fact correct.

Thanks for viewing.

**Comment by dblue** [comment]
I think bibox.js needs to be updated to include the v2 endpoints.

```ccxt.base.errors.ExchangeError: bibox {"error":{"code":"3011","msg":"接口不支持apikey请求方式"}}```

The 3011 error code means "Interface does not support apikey request method" (see https://github.com/Biboxcom/API_Docs_en/wiki/REST_error_code).

You posted to ```https://api.bibox.com/v1/transfer``` but looking at the API, it looks like you should be posting to ```https://api.bibox.com/v2/assets/transfer/spot``` (see https://github.com/Biboxcom/API_Docs_en/wiki/REST_API_Reference#transfer-assets).

**Comment by karlandoh** [comment]
Thats fair, but how exactly do I do that??
```
bibox.privatePostTransfer({'currency': 'BTC', 'amount': 3.685e-05, 'type': 1, 'version': 'v2'})
```
I tried to include the version in my arguments.

**Comment by kroitor** [comment]
> I tried to include the version in my arguments, but with no avail.

@karlandoh we will have to add the necessary v2 endpoints before those can be used with bibox, unfortunately, you cannot override the API version via params, as this requires some more code in CCXT, which is not there yet. We will do our best to add those endpoints as soon as we can. Hope for your understanding! Thx!

**Comment by dblue** [comment]
@kroitor It appears to be a single endpoint and is slightly different from the V1 signatures. I've got it working but I won't get to a clean PR until tomorrow.

**Comment by karlandoh** [comment]
Sounds good, thanks.

**Comment by dblue** [comment]
@karlandoh Here are the patch and diff files to the pull request if you find them useful.

https://patch-diff.githubusercontent.com/raw/ccxt/ccxt/pull/5933.diff
https://patch-diff.githubusercontent.com/raw/ccxt/ccxt/pull/5933.patch

**Comment by karlandoh** [comment]
Looks good, thanks for your help.

**Comment by kroitor** [comment]
@dblue thank you! Merged #5933!
